R/new_city.R

```diff
@@ -48,9 +51,16 @@
 #' }
 #' @export
 
-new_city <- function(name = NULL, country = NULL, lat = NULL, long = NULL) {
-  stopifnot("specify all input arguments" = all(c(!is.null(name), !is.null(country), !is.null(lat), !is.null(long))))
-  stopifnot("all input must be of length 1" = all(c(length(name) == 1, length(country) == 1, length(lat) == 1, length(long) == 1)))
+new_city <- function(name = NULL, country = NULL, lat = NULL, long = NULL, method = "osm") {
```

Comment:
`method = c("osm", "census", "arcgis", "census_simple", "geocodio",
                                           "mapbox", "google", "bing", "here", "tomtom", "nominatim", "tiger")`

Reply:
What do you want here?

Reply:
ok linked to following point